.gitignore

```diff
@@ -1 +1,2 @@
 node_modules
+seedData.txt
```

Comment:
why do your seed data have a txt extension?

Reply:
That's not the actual data. It's just pri's list copied from slack.